corehq/apps/domain/forms.py

```diff
@@ -505,7 +505,7 @@ def save(self, request, domain):
         domain.hr_name = self.cleaned_data['hr_name']
         domain.project_description = self.cleaned_data['project_description']
         domain.default_mobile_ucr_sync_interval = self.cleaned_data.get('mobile_ucr_sync_interval', None)
-        domain.default_geocoder_location = self.cleaned_data['default_geocoder_location']
+        domain.default_geocoder_location = self.cleaned_data.get('default_geocoder_location')
```

Comment:
I don't think this is necessary, due to the other change, but it isn't harmful either.